clients/apple/Shared/Stateful Logic/DocumentService.swift

```diff
@@ -282,7 +282,16 @@ class DocumentLoadingInfo: ObservableObject {
             DispatchQueue.main.async {
                 switch operation {
                 case .success(let txt):
-                    self.textDocument = EditorState(text: txt, isiPhone: self.isiPhone)
+                    self.textDocument = EditorState(text: txt, isiPhone: self.isiPhone) { url in
+                        DI.importExport.importFilesSync(sources:[url.path(percentEncoded: false)], destination: self.meta.parent)
+                        
+                        if let parentPath = DI.files.getPathByIdOrParent(maybeId: self.meta.parent) {
+                            
+                            return "lb:/\(parentPath)\(url.lastPathComponent)"
```

Comment:
two slashes?

Reply:
not sure where exactly this belongs but looking for something like this
```suggestion
                            return "![alt text](lb://\(parentPath)\(url.lastPathComponent))"
```

Reply:
You're right, changed it to the image markdown syntax.